src/views/Transcript/Components/TranscriptActivity/index.js

```diff
@@ -3,9 +3,14 @@ import Divider from 'material-ui/Divider/Divider';
 import React, { Component } from 'react';
 import List from 'material-ui/List';
 import Typography from 'material-ui/Typography';
-import './activityList.css';
+import './transcriptActivity.css';
 
-export default class Activities extends Component {
+//This components is a child of the Transcript component. Seperates Headings from content in order
```

Comment:
Much improved, thank you.

"This component" (no "s"). 
"Separates" (not "Seperates")